config/locales/idv/en.yml

```
@@ -195,6 +194,13 @@ en:
         code below. </p><p>If your letter hasn’t arrived yet, please be patient
         as letters take up to <strong>10 days</strong> to arrive. Thank you for
         your patience.</p>'
+      request_another_letter:
+        button: Request another letter
+        instructions_html: Request a new letter if you have issues with your current
+          letter or it never arrived. Letters take <strong>5 to 10</strong> days
```

Emphasis includes days in all languages.
```suggestion
          letter or it never arrived. Letters take <strong>5 to 10 days</strong>
```